plugins/org.komodo.eclipse.workspace.ui/src/org/komodo/eclipse/workspace/ui/common/util/CoreStringUtil.java

```diff
@@ -0,0 +1,1139 @@
+/*
```

Comment:
This should be unnecessary due to StringConstants. Please update the latter if constants are missing rather than creating a new class